Elmah.Io.Extensions.Logging/ElmahIoLoggerProvider.cs

```diff
@@ -7,16 +7,41 @@ public class ElmahIoLoggerProvider : ILoggerProvider
     {
         private readonly string _apiKey;
         private readonly Guid _logId;
+        private readonly FilterLoggerSettings _filter;
 
-        public ElmahIoLoggerProvider(string apiKey, Guid logId)
+        public ElmahIoLoggerProvider(string apiKey, Guid logId, FilterLoggerSettings filter = null)
         {
             _apiKey = apiKey;
             _logId = logId;
+
+            if (filter == null)
+            {
+                filter = new FilterLoggerSettings
+                {
+                    {"*", LogLevel.Trace}
+                };
+            }
+            _filter = filter;
+        }
+
+        private LogLevel FindLevel(string categoryName)
```

Comment:
Too bad we don't have a test project yet. I'll create that asap and create some unit tests of this.